src/plugins/discover/public/application/main/data_fetching/update_search_source.ts

```diff
@@ -48,7 +50,7 @@ export function updateVolatileSearchSource(
 
   if (dataView.type !== DataViewType.ROLLUP) {
     // Set the date range filter fields from timeFilter using the absolute format. Search sessions requires that it be converted from a relative range
-    const timeFilter = data.query.timefilter.timefilter.createFilter(dataView);
+    const timeFilter = data.query.timefilter.timefilter.createFilter(dataView, inputTimeRange);
```

Comment:
I think we need to be passing a time range to the histogram which is a result of `createFilter` instead of the input time range because the bounds can be different due to its internal logic.